### ISSUE TITLE
Accessibility: input does not have an associated label

### ISSUE THREAD

**Issue by tartan75** [viclafouch/mui-file-input#53]
**Describe the bug**
Axe accessibility testing reports the error "Form elements must have labels". While there is a label the for attrible has a differant ID to that of the input.

**To Reproduce**
1. Follow repo storybook example but without placeholder prop (as that prop stops Axe with another warning)

**Expected behavior**
The input should have a valid label for accessibility WCAG2A, and other standards

**Additional context**
The issue appears to be the result of the id of the input being set on input.tsx line 14. A change to `    const id = props.id` appears to resolve the issue, however I have not checked if there are adverse effects from that change.

**Comment by viclafouch** [comment]
Hello !

Happy to open a PR ;)
